Complex Queries - Day Four

tinatsina opened the pull request:
## Tasks achieved 🥇 
- added new tables to `schema.sql`
- added new entries to `data.sql`
- added new queries to `queries.sql`

## Screenshots 🎥 
![image](https://github.com/tinatsina/vet_clinic_db/assets/44028673/1dc7494d-cb79-428d-8c93-c295de38f5d2)
![image](https://github.com/tinatsina/vet_clinic_db/assets/44028673/4901e313-14e1-43d0-bd86-afb4f2f0a726)
![image](https://github.com/tinatsina/vet_clinic_db/assets/44028673/cee79a45-43ca-492c-8fdc-57cba5821f9a)
![image](https://github.com/tinatsina/vet_clinic_db/assets/44028673/ab67207b-8a13-4457-a16a-3f28549a582e)
![image](https://github.com/tinatsina/vet_clinic_db/assets/44028673/4cb49561-da59-41d0-af3b-5e072e101c0c)
![image](https://github.com/tinatsina/vet_clinic_db/assets/44028673/0ef40db0-aa8f-490e-b792-b655894bf37e)
![image](https://github.com/tinatsina/vet_clinic_db/assets/44028673/9a7c6da5-f5ba-4463-869a-07be18aeb27a)

Shedrack-Sunday reviewed (changes requested):
Hello @tinatsina          👋 

# CHANGES REQUIRED ♻️

Impressive work on the project so far👏
You've done extremely well so far in this project,  However, there are some adjustments you need to make to move to the next project. 💯

![](https://user-images.githubusercontent.com/80612925/157397398-00904e26-60ec-46cd-9510-240661c250ef.gif)

## Highlights   
- Good use of GitHub features ✔️   
- Nice work with the SQL features ✔️ 
- Nice use of query parameters✔️ 

# Required Changes  🌵  

- [ ] You've done well, However, kindly follow the design guidelines properly so your impressive work will have a better user experience 👍 

- [ ] Your MIT file is not leading to the right path. Kindly include the right file name in the root repository so its clickable 

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

tinatsina commented:
## Review changes resolved
- added a new LICENSE file
- The MIT LICENSE file is now available
- Also changed the design for the better implementation

Abenezer-Tilahun reviewed (approved):
Hi @tinatsina 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.


## Optional suggestions 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
